### PR TITLE
feat: re-add in-memory subscriptions loader

### PR DESCRIPTION
**Pull request by subomi**

### CODE REVIEW

File: database/postgres/subscription.go [hunk 1]
```diff
@@ -121,6 +122,40 @@ const (
...
+
+	fetchDeletedSubscriptions = `
+    select name, id, type, project_id, endpoint_id, function, updated_at,
+    filter_config_event_types AS "filter_config.event_types",
+    filter_config_filter_headers AS "filter_config.filter.headers",
+	filter_config_filter_body AS "filter_config.filter.body"
+    from convoy.subscriptions
+    where (deleted_at > $4 AND deleted_at is not null)
+    AND id > $1
+    AND project_id = $2
```
Comment (danvixent) [inline database/postgres/subscription.go:158]: @subomi please make a base fetch constant query, that way we can avoid any one of them just randomly changing.

File: internal/pkg/memorystore/table.go [hunk 2]
```diff
@@ -85,18 +85,34 @@ func (t *Table) Add(key string, value interface{}) *Row {
...
 
+// Add a new item if it doesn't exist.
+func (t *Table) Upsert(key string, value interface{}) *Row {
+	t.Lock()
+	defer t.Unlock()
+
+	row := t.getInternal(key)
+	if row != nil {
+		t.deleteInternal(key)
+	}
```
Comment (danvixent) [inline internal/pkg/memorystore/table.go:119]: The row key will be updated or created by the t.rows map, extra check not needed.

File: database/postgres/subscription.go (hunk 1 above)
Comment (subomi) [reply database/postgres/subscription.go:158]: @danvixent I'll defer this for a later refactor.

File: worker/task/process_broadcast_event_creation.go [hunk 3]
```diff
@@ -76,6 +76,7 @@ func ProcessBroadcastEventCreation(endpointRepo datastore.EndpointRepository, ev
 		es, ss := getEndpointIDs(subscriptions)
 		event.Endpoints = es
 
+		fmt.Println(es)
```
Comment (jirevwe) [inline worker/task/process_broadcast_event_creation.go:79]: Remove print

**Review by danvixent — APPROVED** [review]

**Pull request merged**